src/main/java/com/snowflake/kafka/connector/records/RecordService.java

```diff
@@ -54,7 +54,7 @@
 
   public static final SimpleDateFormat ISO_DATE_FORMAT= new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
   public static final SimpleDateFormat TIME_FORMAT= new SimpleDateFormat("HH:mm:ss.SSSZ");
-  private static final int MAX_SNOWFLAKE_NUMBER_PRECISION = 38;
+  public static final int MAX_SNOWFLAKE_NUMBER_PRECISION = 38;
```

Comment:
Moving this parameter to public for test to access.

Reply:
package private?

Reply:
ack